service/quote-server/tdx_lib/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "tdx_lib"
```

Comment:
How about changing the name to `tdx_attest`, and we can submit this library to crates.io.

Reply:
that's a good idea, I've add an issue to track this: https://github.com/intel/confidential-cloud-native-primitives/issues/107